src/Raven.Server/Documents/Handlers/Debugging/NodeDebugHandler.cs

```diff
@@ -158,7 +158,7 @@ private async Task<PingResult> PingOnce(string url)
 
             using (var cts = new CancellationTokenSource(ServerStore.Engine.TcpConnectionTimeout))
             {
-                var info = await ReplicationUtils.GetTcpInfoAsync(url, null, "PingTest", ServerStore.Engine.ClusterCertificate, cts.Token);
+                var info = await ReplicationUtils.GetTcpInfoAsync(url, null, "PingTest", ServerStore.Engine.ClusterCertificate, ServerStore.NodeTag, cts.Token);
```

Comment:
why?

Reply:
```GetTcpInfoAsync``` now sends the source node tag for checking in the other side (dest) if it's rehab in it's topology.

Reply:
Removed